internal/cli/client/client.go

```diff
@@ -26,7 +26,12 @@ func GetClientFromConfig(ctx context.Context, opt *option.Option) (
 	}
 	cfg, err := config.LoadCLIConfig()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "load cli config")
+	}
+	if cfg.APIAddress == "" || cfg.BearerToken == "" {
+		return nil, errors.New(
+			"seems like you are not logged in; please use `kargo login` to authenticate",
+		)
 	}
```

Comment:
This condition will never occur. The previous action was to load the config, and that will have failed with an error if it didn't exist.

I think you can safely remove this condition for now.

Later, we should revisit the fact that config is being re-loaded when building a client. We shouldn't be doing that, but that is something we can deal with later.

Reply:
No, it can occur by removing `apiAddress` or `bearerToken` field in Kargo config file.

```
❯ cat ~/.config/kargo/config            
project: kargo-demo

❯ ./cli get projects
Error: get client from config: seems like you are not logged in; please use `kargo login` to authenticate
```

Reply:
Oh. I see.